actual/umbrel-app.yml

```
@@ -0,0 +1,33 @@
+manifestVersion: 1
+id: actual
+category: finance
+name: Actual
+version: "25.5.0"
+tagline: Powerful budgeting made simple
+description: >-
+  Actual Budget is a super fast and privacy-focused app for managing your finances. At its heart is the well proven and much loved Envelope Budgeting methodology. You own your data and can do whatever you want with it. Featuring multi-device sync, optional end-to-end encryption and so much more.
+
+  - Automated finance tools are great, except when they aren’t. We provide you with tools that are quick to use, but ultimately you are in control. We help you learn, instead of dictating.
+
+  - A beautifully designed interface is fine-tuned to get out of your way and make it as fast as possible to explore your finances.
+
+  - Actual is a local app, plain and simple. Your data is synced in the background so all devices have access, but the app totally works regardless of your network connection. This also allows end-to-end encryption to keep your data private.
+
+  - Save hundreds of dollars a year (at least!) by tracking your spending. Based on tried and true methods, our budgeting system is based off of your real income instead of made up numbers. This makes you face your real spending, and clearly shows how much you are saving each month. We make this process as simple as possible. 
+
+  - Breeze through your transactions and update them easily with a streamlined, minimal interface. Categorizing your transactions correctly is important and we’ve optimized this process. Manage split transactions and transfers all in the same editor.
+  
+  - Intuitive reports give you a quick way to learn about your finances. By default, we include net worth and cash flow reports. Actual also includes a powerful custom report engine to design your own reports to fit your needs.
+developer: actualbudget
+website: https://actualbudget.org/
+repo: https://github.com/actualbudget/actual
+support: https://github.com/actualbudget/actual/issues
+port: 5006
+submitter: Abhiraj Shourya
+submission: https://github.com/getumbrel/umbrel-apps/pull/2827
+gallery: []
+dependencies: []
+path: ""
+defaultUsername: ""
+defaultPassword: ""
+releaseNotes: ""
```

Add a final newline.
```suggestion
releaseNotes: ""

```